src/css/app.scss

```diff
@@ -356,3 +356,11 @@ footer {
         }
     }
 }
+
+ul.unstyled-list {
+    padding-inline-start: 0;
```

Comment:
Historically the cross-browser thing to do here was to zero out margin and padding for `ul` and `li`. This may be enough nowadays, not sure.

That said, let's attach these rules to `ul.wallets` and drop the extraneous class. If we want to share the definitions later we can use a [mixin](https://sass-lang.com/documentation/at-rules/mixin) or an `@extend` [placeholder selector](https://sass-lang.com/documentation/at-rules/extend#placeholder-selectors).